scrapers/capital.py

```diff
@@ -38,12 +38,16 @@ def scrape_page(self):
                     'title': text
                 })
             print (json.dumps(data, indent=2))
-            self.base.s3.put_object(
-                Bucket='taxclock.codeforkenya.org',
-                ACL='public-read',
-                Key='data/standard-news.json',
-                Body=json.dumps(data)
-            )
-            return result_html
+            try:
+                self.base.s4.put_object(
+                    Bucket='taxclock.codeforkenya.org',
+                    ACL='public-read',
+                    Key='data/standard-news.json',
+                    Body=json.dumps(data)
+                )
+            except Exception as err:
+                self.base.local_store(data)
+                log.info(str(err))
+            return data
```

Comment:
This is repeating through all the scrapers. Should it not be a function of base scraper instead?